index.js

```diff
@@ -72,8 +72,14 @@ module.exports = function childrenOfPid(pid, includeRoot, callback) {
       }
 
       // Convert RSS to number of bytes
-      columns[4] = parseInt(columns[4], 10);
-      if (process.platform !== 'win32') {
+      if (process.platform == 'win32') {
+          // For Windows, WMIC.exe never returns any value for "Status" and it causes "WorkingSetSize" is set at columns[3]
+          // See: https://docs.microsoft.com/ja-jp/windows/win32/cimwin32prov/win32-process?redirectedfrom=MSDN
+          columns[4] = parseInt(columns[3], 10);
```

Comment:
It would be safer to just skip "Status" in the WMIC call above. In case WMIC shows some non-empty string for it, this code here will fail.